Source/render/ShapeRenderer.swift

```diff
@@ -32,14 +32,53 @@ class ShapeRenderer: NodeRenderer {
     }
 
     override func doRender(_ force: Bool, opacity: Double) {
-        guard let shape = shape else {
-            return
+        guard let shape = shape, let context = ctx.cgContext else { return }
+        
+        if let blur = shape.effect as? GaussianBlur {
+            let shadowInset = min(blur.radius * 6 + 1, 150)
```

Comment:
Looks like magic 😳Where these constants came from?

Reply:
My head( I didn't find a simple explanation of how to calculate it. Taking blur.radius is not nearly enough, and anything more than 150 is cut in firefox